### PR TITLE
Feature/share buttons

### PR DESCRIPTION
**Pull request by thomas-desmond**
Adding a "Share" button to the outcome page that brings up a modal with a copy-able URL and share buttons for popular social media platforms.

### CODE REVIEW

**Comment by thomas-desmond** [comment]
Matched ordering and feel of Dev Portal Share Icons (https://developers.sitecore.com/changelog)

Viewable Link: https://sitecore-migration-game-git-f-de9894-sitecoretechnicalmarketing.vercel.app/outcome/48fc3ad8-ecc4-4a53-a07b-2cd136a59959

File: cypress/e2e/OutcomeFlow/outcome.cy.ts
Comment (dylanyoung-dev) [inline cypress/e2e/OutcomeFlow/outcome.cy.ts:1]: I wonder if combining multiple things in a test is a good idea.  For now just a generalized comment.  Nothing to change I don't think.  But I was wondering how you might use before (which could run the steps to get to the outcome page) and then have individual tests for each of the cases you want to test.  Not sure that's how that feature works lol

**Review by dylanyoung-dev — APPROVED** [review]
Will say, I love that we can review specific outcome pages.  Also like the messaging.  Would be cool if we can play with tweaking the messaging in the future, so that we could see what messages engage the best. (But obviously out of scope for this item).

So just one short comment about the test, but I don't think that's necessary to change.  So looks good.

File: cypress/e2e/OutcomeFlow/outcome.cy.ts
Comment (dylanyoung-dev) [reply cypress/e2e/OutcomeFlow/outcome.cy.ts:1]: Definitely don't want to run the same flow multiple times to test different things.  So I'm not sure that which I thought of works where it only runs the test once up to the outcome page, but maybe something to look into?

Comment (thomas-desmond) [reply cypress/e2e/OutcomeFlow/outcome.cy.ts:1]: I agree. We should have a few full-flow tests from beginning to end, but I don't want one test to end up testing 15 things working on the outcome page. 

I think now that we have outcome pages we could visit the URL  `/outcome/48fc3ad8-ecc4-4a53-a07b-2cd136a59959` as the first step of the cypress test. We don't need to go through the whole flow for certain things like the share button. 

Merging this for now but any further outcome page tests we should consider just navigating directly to a known existing outcome.

**Pull request merged**